src/M6Web/Bundle/LogBridgeBundle/Matcher/Status/Type/CompleteType.php

```diff
@@ -26,7 +28,7 @@ protected function getPattern()
      */
     protected function transform($config)
     {
-        $refStatus = substr($config, 0, strlen((int) $config - 2));
+        $refStatus = substr($config, 0, strlen((string) ((int) $config - 2)));
```

Comment:
**question (non-blocking):** why did you need to do this? 🤔

Reply:
I changed the version of `m6web/php-cs-fixer-config` and it now adds this line `declare(strict_types=1);` at the beginning of each file. Because of this, an error was raised on this line because `strlen` expects its first argument to be a string.
I have to admit that I don't really understand what this line does and I didn't want to break anything so I went with this 🤷

Reply:
I tried it [online](https://3v4l.org/CK1RM) and it does not break the current goal of this statement 
(which seems to be counting the number of digits of an integer)